ext/pdo_pgsql/pdo_pgsql.c

```diff
@@ -156,6 +156,9 @@ PHP_MINIT_FUNCTION(pdo_pgsql)
 	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_TRANSACTION_INTRANS", (zend_long)PGSQL_TRANSACTION_INTRANS);
 	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_TRANSACTION_INERROR", (zend_long)PGSQL_TRANSACTION_INERROR);
 	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_TRANSACTION_UNKNOWN", (zend_long)PGSQL_TRANSACTION_UNKNOWN);
+#ifdef HAVE_PG_RESULT_MEMORY_SIZE
+	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_ATTR_RESULT_MEMORY_SIZE", (zend_long)PDO_PGSQL_ATTR_RESULT_MEMORY_SIZE);
+#endif
```

Comment:
I'm not sure if I should write this down.

The purpose of subclasses is to allow driver specific functionality to be provided in the subclasses.

This statement would mean having the same constants in PDO core, which may go against the subclassing philosophy.

Reply:
However, since PDO::getAttribute is not a subclass-specific method, I'm not sure what to do.